lib/const.rb

```diff
@@ -61,7 +61,7 @@
 end
 
 # Use sane minimal defaults if in container and no override specified.
-CREW_KERNEL_VERSION ||= \
+CREW_KERNEL_VERSION ||=
```

Comment:
This is a line continuation indicator and probably should remain.

Reply:
This change is from `Style/RedundantLineContinuation`: https://docs.rubocop.org/rubocop/cops_style.html#styleredundantlinecontinuation

From what I can tell, this is safe to autocorrect and does not change behavior at all.